### PR TITLE
Group2

### PR DESCRIPTION
**Pull request by mceccarelli**
We are almost finished with our section, but here's what we have now.

### CODE REVIEW

**Comment by noorbuchi** [comment]
Can you please update us on the status of this PR. Is it ready for review or is it still a work in progress? Thanks!

**Comment by GabeSchwartz** [comment]
Im still working on it.

**Comment by noorbuchi** [comment]
Thanks for the update! I will label it as a `WIP` (work in progress) and once your group is ready for review, you can remove that label.

**Review by jjumadinova — CHANGES_REQUESTED** [review]
Please merge the latest changes from the main branch.

File: assessment.md
Comment (noorbuchi) [inline assessment.md]: I think it would be great to elaborate on what is the "inappropriate" use of the GitHub flow model. It could be using unclear or inappropriate commit messages, merging PRs without getting approval from TLs and instructor, deleting other groups' branches... etc. Each one of these uses should be somewhat addressed in the assessment criteria,  not just for `Using the GitHub Flow model` but for others as well

File: assessment.md
Comment (noorbuchi) [inline assessment.md]: This section is missing. Please don't forget to add it

File: assessment.md
Comment (noorbuchi) [inline assessment.md]: Again, I think it would be great to elaborate on what is the appropriate and inappropriate use of branches. Things like keeping a branch up to date with `main`, deleting a branch after a PR is merged if it's not needed, appropriately names the branches, committing to the right branch.. etc. These are some suggestions that you could incorporate in the assessment criteria.

File: assessment.md
Comment (noorbuchi) [inline assessment.md]: In many cases, using a fork too often is not always good. Meaning that if you have collaborator access to the project, it would be better to create a branch that all contributors can see and review instead of creating a fork under your own username. You don't have to remove that statement, however, rephrasing it can make more sense.

File: assessment.md
Comment (noorbuchi) [inline assessment.md]: I think you should replace `comments` with `description` since comments usually refer to comments made in each file containing code. Descriptions are more general and they refer to the PR description as well as other discussions that could take place. Finally, it's also worth mentioning merging branches where the continuous integration build might be failing and how that should not be ok.

File: assessment.md
Comment (noorbuchi) [inline assessment.md]: Missing section

File: assessment.md
Comment (noorbuchi) [reply assessment.md]: Looking back at my comment after reading through your addition. It looks like you've addressed some of it but there are still few things missing. I think that giving examples of what is appropriate and inappropriate uses is still needed. Feel free to ask questions if you need me to clarify my change requests. Thanks

**Review by noorbuchi — CHANGES_REQUESTED** [review]

**Review by enpuyou — APPROVED** [review]
Once the team has addressed @noorbuchi's review, I think the PR would be good to go. Thanks!

**Review by corlettim — APPROVED** [review]
As long as you update your branch with master, you are good to go!

**Review by munzekm — APPROVED** [review]

**Review by noorbuchi — APPROVED** [review]

**Comment by noorbuchi** [comment]
Please update the branch with main and this PR should be good to go!

**Pull request merged**
